modules/phase_field/tests/reconstruction/2phase_reconstruction4.i

```diff
@@ -45,7 +45,7 @@
   [./PolycrystalICs]
     [./ReconVarIC]
       ebsd_reader = ebsd
-      advanced_op_assignment = true
+      coloring_algorithm = bt
```

Comment:
Since legacy is the default, it's used in several of the non-graintracker tests. If I switch the default, we'll see this breakage in several places. I don't think we need a test, this is the normal reduced OP stuff we've been doing in marmot for years.